internal/tests/parallel/parallel_test.go

```diff
@@ -33,7 +34,13 @@ func TestTasksWithError(t *testing.T) {
 func TestTasksWithPanic(t *testing.T) {
 	err := TasksWithPanic()
 	require.Error(t, err)
-	assert.Equal(t, "panic: sad times", err.Error())
+	assert.Contains(t, err.Error(), "panic: sad times\nstacktrace:")
+	// check that error returned is actually a panic error
+	var panicError *cff.PanicError
+	assert.Equal(t, true, errors.As(err, &panicError), "error returned should be a cff.PanicError")
```

Comment:
Can we use `require.ErrorsAs`? https://pkg.go.dev/github.com/stretchr/testify/require#ErrorAs
(require since I think the following checks will panic if we don't FailNow during this check)